cdp-agentkit-core/typescript/src/actions/cdp/get_balance_nft.ts

```diff
@@ -0,0 +1,70 @@
+import { CdpAction } from "./cdp_action";
+import { readContract, Wallet } from "@coinbase/coinbase-sdk";
+import { Hex } from "viem";
+import { z } from "zod";
+
+const GET_BALANCE_NFT_PROMPT = `
+This tool will get the NFTs owned by the wallet for a specific NFT contract.
```

Comment:
can we specify this is for ERC-721 contracts like we do in the description for transfer.

Reply:
Updated